src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -162,6 +163,9 @@
 			<x:Double x:Key="MaterialOutlinedTextBoxMinHeight">56</x:Double>
 			<Thickness x:Key="MaterialOutlinedTextBoxBorderPadding">1</Thickness>
 			<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessFocused">2</x:Double>
+			<Thickness x:Key="MaterialOutlinedTextBoxPaddingFocused">0</Thickness>
+			<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessPointerOver">2</x:Double>
+			<Thickness x:Key="MaterialOutlinedTextBoxPaddingPointerOver">0</Thickness>
 			<!--#endregion-->
 		</ResourceDictionary>
 
```

Comment:
@kazo0 do we need to verify Markup as well here ?
https://github.com/unoplatform/Uno.Themes/blob/f4256d1e3fee258d24307d2b8d93ba87f64a6633/src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs